cobrapy does not seem to work with libsbml 5.19

#### Problem description

I'm trying to update the cobrapy package in Debian to be up to date with the latest version in the next Debian stable release.  Debian has libsbml version 5.19.0 packaged.  When building against this version I get a lot of test suite errors that seem to be caused by incompatibilities with sbml.  For completeness and to let you know what versions are used here is a full build log:

[python-cobra_0.20.0-1_amd64.zip](https://github.com/opencobra/cobrapy/files/5867974/python-cobra_0.20.0-1_amd64.zip)

You can find the error messages at the end of the log.

Please let me know if I can provide any more detailed information.

Kind regards, Andreas.

I think the errors are actually related to https://github.com/biosustain/swiglpk/issues/39. The SBML error is only since it is kind of a catch all for exceptions when loading the textbook model which most tests do. Further down you can find the swiglpk errors:

```
...
    from sys import version_info as _swig_python_version_info
    if _swig_python_version_info < (2, 7, 0):
        raise RuntimeError("Python 2.7 or later required")
    
    # Import the low-level C/C++ module
    if __package__ or "." in __name__:
>       from . import _swiglpk
E       ImportError: /usr/lib/python3/dist-packages/swiglpk/_swiglpk.cpython-39-x86_64-linux-gnu.so: undefined symbol: glp_netgen_prob
...
```

Due to that bug cobrapy is currently not compatible with Python 3.9. I submitted a fix in https://github.com/biosustain/swiglpk/pull/42 a while ago but the maintainer has not responded yet.

--- a/release-notes/next-release.md
+++ b/release-notes/next-release.md
@@ -4,6 +4,9 @@
 
 ## Fixes
 
+## Other
+* Update dependency on python-libsbml-experimental to >=5.19.0.
+
 ## Deprecated features
 
 ## Backwards incompatible changes

--- a/setup.cfg
+++ b/setup.cfg
@@ -6,7 +6,7 @@ parse = (?P<major>\d+)
 	\.(?P<minor>\d+)
 	\.(?P<patch>\d+)
 	(?P<release>[a]*)(?P<num>\d*)
-serialize = 
+serialize =
 	{major}.{minor}.{patch}{release}{num}
 	{major}.{minor}.{patch}
 tag_name = {new_version}
@@ -15,7 +15,7 @@ tag_name = {new_version}
 name = cobra
 url = https://opencobra.github.io/cobrapy
 download_url = https://pypi.org/project/cobra
-project_urls = 
+project_urls =
 	Source Code = https://github.com/opencobra/cobrapy
 	Documentation = https://cobrapy.readthedocs.io
 	Bug Tracker = https://github.com/opencobra/cobrapy/issues
@@ -23,7 +23,7 @@ author = The cobrapy core development team.
 author_email = cobra-pie@googlegroups.com
 maintainer = Moritz E. Beber
 maintainer_email = moritz.beber@gmail.com
-classifiers = 
+classifiers =
 	Development Status :: 5 - Production/Stable
 	Intended Audience :: Science/Research
 	License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)
@@ -39,7 +39,7 @@ license = LGPL-2.0-or-later OR GPL-2.0-or-later
 description = COBRApy is a package for constraint-based modeling of metabolic networks.
 long_description = file: README.rst, INSTALL.rst
 long_description_content_type = text/x-rst
-keywords = 
+keywords =
 	metabolism
 	biology
 	constraint-based
@@ -51,7 +51,7 @@ keywords =
 
 [options]
 zip_safe = True
-install_requires = 
+install_requires =
 	appdirs ~=1.4
 	depinfo
 	diskcache ~=5.0
@@ -62,29 +62,29 @@ install_requires =
 	optlang <1.4.6
 	pandas ~=1.0
 	pydantic ~=1.6
-	python-libsbml-experimental ==5.18.3
+	python-libsbml-experimental >=5.19.0
 	rich ~=6.0
 	ruamel.yaml ~=0.16
 	six
 	swiglpk
-tests_require = 
+tests_require =
 	tox
 packages = find:
-package_dir = 
+package_dir =
 	= src
 
 [options.packages.find]
 where = src
 
 [options.package_data]
-cobra = 
+cobra =
 	io/*.json
 	test/data/*
 
 [options.extras_require]
-array = 
+array =
 	scipy
-development = 
+development =
 	black
 	bumpversion
 	isort
@@ -96,7 +96,7 @@ universal = 1
 [bumpversion:part:release]
 optional_value = placeholder
 first_value = placeholder
-values = 
+values =
 	placeholder
 	a
 

